cmd/singularity/actions_test.go

```diff
@@ -176,6 +227,9 @@ func testSTDINPipe(t *testing.T) {
 		{"sh", "library", []string{"-c", "echo true | singularity shell library://busybox"}, 0},
 		{"sh", "docker", []string{"-c", "echo true | singularity shell docker://busybox"}, 0},
 		{"sh", "shub", []string{"-c", "echo true | singularity shell shub://singularityhub/busybox"}, 0},
+		// Test a Bad target pwd
+		{"sh", "pwdPath", []string{"-c", fmt.Sprintf("singularity exec --pwd /etc %s pwd | egrep '^/etc'", imagePath)}, 0},
+		{"sh", "falsePwdPath", []string{"-c", fmt.Sprintf("singularity exec --pwd /bad/path %s pwd | egrep '^/etc'", imagePath)}, 1},
```

Comment:
Why are you looking for `/etc` here? Shouldn't it be more like:
```
{"sh", "falsePwdPath", []string{"-c", fmt.Sprintf("singularity exec --pwd /bad/path %s pwd | egrep '^/bad/path'", imagePath)}, 1}
```
Look for what you used on `--pwd`, and expect a failure?